src/components/Data/SourceChart.js

```diff
@@ -22,7 +22,7 @@ const SourceChart = ({loveFunction}) => {
   const hearts = loveFunction(4);
   return(
   <div>
-    <h5 className="mb-4">`Developer Love: ${hearts}`</h5>
+    <h5 className="mb-4">Developer Love: {hearts}</h5>
```

Comment:
```suggestion
    <h5 className="mb-4">{`Developer Love: ${hearts}`}</h5>
```

This will solve the problem

Reply:
Actually this will not work, it will return something like:
Developer Love: [object],[object]

Instead I would replace line 25 with:
`
  <Tooltip tooltipText='This is a score out of 5 based on developer opinion, community size, downloads, google searches, and satisfaction surveys, etc..'>
  <h5 className="pr-1">Developer Love:</h5>
  <h5 className="pl-1">{hearts}</h5>
</Tooltip>
`
Making sure to include tooltip competent. This will make it the same as the top half of the page